plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/_impl.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import (
+    Any,
+    Dict,
+    List,
+    MutableMapping,
+    MutableSequence,
+    Optional,
+    Tuple,
+    Union,
+)
+
+import nevergrad as ng
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.override_parser.overrides_parser import OverridesParser
+from hydra.core.override_parser.types import (
+    ChoiceSweep,
+    IntervalSweep,
+    Override,
+    Transformer,
+)
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+from hydra.plugins.sweeper import Sweeper
+from hydra.types import TaskFunction
+from omegaconf import DictConfig, ListConfig, OmegaConf
+
+from .config import OptimConf, ScalarConfigSpec
+
+log = logging.getLogger(__name__)
+
+
+def create_nevergrad_param_from_config(
+    config: Union[MutableSequence[Any], MutableMapping[str, Any]]
+) -> Any:
+    if isinstance(config, (list, ListConfig)):
```

Comment:
Does this work now?
```suggestion
    if isinstance(config, MutableSequence):
```

Reply:
yea, it worked (tests passed and mypy didnt complain)

thx for catching them, fixed now.